lib/console.js

```diff
@@ -166,6 +169,42 @@ Console.prototype.assert = function assert(expression, ...args) {
   }
 };
 
+// Defined by: https://console.spec.whatwg.org/#clear
+Console.prototype.clear = function clear() {
+  // It only makes sense to clear if _stdout is a TTY.
+  // Otherwise, do nothing.
+  if (this._stdout.isTTY) {
+    // The require is here intentionally to avoid readline being
+    // required too early when console is first loaded.
+    const { cursorTo, clearScreenDown } = require('readline');
+    cursorTo(this._stdout, 0, 0);
+    clearScreenDown(this._stdout);
+  }
+};
+
+// Defined by: https://console.spec.whatwg.org/#count
+Console.prototype.count = function count(label = 'default') {
+  // Ensures that label is a string, and only things that can be
+  // coerced to strings. e.g. Symbol is not allowed
+  label = `${label}`;
+  const counts = this[kCounts];
+  let count = counts.get(label);
+  if (count === undefined)
+    count = 1;
+  else
+    count++;
+  counts.set(label, count);
+  this.log(`${label}: ${count}`);
+};
+
+// Not yet defined by the https://console.spec.whatwg.org, but
+// proposed to be added and currently implemented by Edge. Having
+// the ability to reset counters is important to help prevent
+// the counter from being a memory leak.
+Console.prototype.countReset = function countReset(label = 'default') {
+  const counts = this[kCounts];
+  counts.delete(label);
```

Comment:
No, I don't think so. It's an idempotent operation that can simply ignore invalid input.